qemu/tests/cfg/nested_test.cfg

```diff
@@ -1,6 +1,7 @@
 - nested_test:
     kar_repo = <kar_repo_url>
     cert_url = <cert_ca_url>
+    bootstrap_options = ""
```

Comment:
This is for bootstrap in L1 guest, how about `nested_bs_options` ?  To avoid host bootstrap use such kind of options.